app/lib/validation_utils.rb

```diff
@@ -2,4 +2,12 @@ module ValidationUtils
   def valid_year?(year)
     year.to_s.match?(/^[12]\d{3}$/)
   end
+
+  def future_year?(year)
+    year.to_i >= year_limit
+  end
+
+  def year_limit
+    Time.zone.today.year.to_i + 1
+  end
```

Comment:
Suggest simplifying this as:

```suggestion
  def future_year?(year)
    year.to_i >= Time.zone.today.year
  end
```

Reply:
Hey Frankie, unfortunately this won't work because `year_limit` needs to be a separate method as it is called by the error message in the form's validation method as well. Happy to discuss.

Reply:
Sorry, I forgot to batch up this comment with the next one...

Let's talk to a content designer.

Reply:
Have implemented in accordance with below change to error message